index.html

```diff
@@ -0,0 +1,393 @@
+                                    </div>
+
+
+                                    <div class="col-4">
+                                        <h3>Innovation</h3>
+                                    </div>
+
+                                    <div class="col-5">
+                                        <p>Even though the pandemic has had negative effects on the economy, it has also created new opportunities for innovating.</p>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-11">
+                                <div class="row card-contents">
+
+                                    <div class="col-3">
+                                        <img src="images/program_icon_05.png" alt="">
+
+                                    </div>
+
+                                    <div class="col-4">
+                                        <h3>Support</h3>
+                                    </div>
+
+                                    <div class="col-5">
+                                        <p>Governments in Africa have been investing in technology-based initiatives to support the development of the sector.</p>
+                                    </div>
+                                </div>
+                            </div>
+
+                        </div>
+                        <div class="row">
+                            <div class="col-1"></div>
+                            <div class="col-10 join_summit">
+                                <center>Join summit 2024</center>
+                            </div>
+
+                        </div>
+
+                    </div>
+
+
+                </div>
+                <div class="row down_part">
+                    <div class="col-12">
+
+                        <center>
+                            <span>
+                                <a href="##">See whole program </a>
+                            </span>
+                        </center>
+
+                    </div>
+                </div>
+            </section>
+
+
+
+
+            <!--Down section-->
+            <section class="container down_section" id="desktop_speakers">
+
+      <!--contents of speaker cards coming from DOM in javascript-->
+                
+
+            </section>
+            <!--Mobile speaker parts-->
+            <!--Down section-->
+            <section class="container down_section" id="mobile_skpeakers">
+
+
+                    
+            </section>
+            <!-- partners -->
+            <div class="w-100 partners">
+                <img src="./images/partners.png" alt="">
+
+            </div>
+
+            <!-- footer -->
+            <div class="container footer_home">
+                <div class="row">
+                    <div class="col-2"></div>
+                    <div class="col-2 footer-logo-1">
+                        <img src="./images/women.png" alt="">
```

Comment:
- [ Required ] Please consider adding a descritptive value for the  `alt` attribute for this image, a descriptive `alt` attribute will make your web page accessible to persons using web accessibility tools like screen readers.